src/debian/stretch-slim/docker-testrunner/arm64v8/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Dockerfile used to create a testrunner image that can perform Docker operations.
+# Usage:  docker run --rm -v /var/run/docker.sock:/var/run/docker.sock testrunner pwsh -File xyz.ps1
+
+FROM arm64v8/debian:stretch-slim
+
+# Install Docker
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        apt-transport-https \
+        ca-certificates \
+        curl \
+        gnupg2 \
+        libicu57 \
```

Comment:
Why is this required in the arm64 variant but not x64?

Reply:
It is required.  But since the x64 version installs PowerShell via a package, libicu gets installed as a dependency automatically.  In the arm64 variant, we're installing PowerShell manually, so we need to explicitly include libicu.